docs/installation.md

```diff
@@ -30,6 +30,12 @@ Determine your base installation option.
 * A **namespace install** only executes workflows in the namespace it is installed in (typically `argo`). Look for `namespace-install.yaml` in the [release assets](https://github.com/argoproj/argo-workflows/releases/latest).
 * A **managed namespace install**: only executes workflows in a specific namespace ([learn more](managed-namespace.md)).
```

Comment:
```suggestion
* A **managed namespace install**: only executes workflows in a separate namespace from the one it is installed in See [Managed Namespace](managed-namespace.md) for more details.
```

Reply:
Thank you for your detailed feedback  @agilgur5! I applied your suggestion, so result are as follows.

This is `managed-namespace.md`
<img width="749" alt="image" src="https://github.com/argoproj/argo-workflows/assets/39519869/38d2d509-f06a-4167-83d7-7607151dbdff">

This one is changed description with link of managed-namespace.md in `installation.md`
<img width="698" alt="image" src="https://github.com/argoproj/argo-workflows/assets/39519869/c5deaf4e-4164-4ba6-93b9-accbef1f6963">

Reply:
Awesome, looks great and thanks for providing the screenshots too!